src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -557,6 +562,9 @@ private function registerTemplatingConfiguration(array $config, $ide, ContainerB
                 $container->removeDefinition('templating.helper.assets');
             }
         }
+
+        $definition = $container->getDefinition($aliased);
```

Comment:
This has to be done in a compiler pass. The `templating.engine.twig` definition, for example, is registered in the TwigBundle and thus not available here.